_includes/jobs/item.html

```diff
@@ -4,4 +4,5 @@ <h5 class="mb-1">{{ page.title }}</h5>
     <small><date title="{{ page.posted_at | date: "%d/%m/%Y" }}" class="timeago">{{ page.posted_at }}</date></small>
   </div>
   <small>{{ page.role }} @ {{ page.company }}</small>
+    <span class="badge badge-secondary">{{ page.tag }}</span>
```

Comment:
Bom dia @guilhermeorcezi

Este trecho está ótimo, porém precisamos de duas coisas aqui...

1. A propriedade `page.tags` é uma lista, e por tanto precisamos iterar para gerar as `n vezes` essa html tag para cada item

Para fazer isso, você pode percorrer a lista de tags usando o [for](https://shopify.github.io/liquid/tags/iteration/#for)... algo mais ou menos como o código abaixo.

```suggestion
    {% for tag in page.tags %}
      <span class="badge badge-secondary">{{ tag }}</span>
    {% endfor %}
```

2. Adicionando apenas as tags elas ficariam na frente do nome da empresa e a ideia é que fiquem embaixo.

Por isso a dica aqui é adicionar esse `for` dendo de um div que já deixará o alinhamento correto.

```liquid
<small>{{ page.role }} @ {{ page.company }}</small>
<div>
  {% for tag in page.tags %}
    ...
  {% endfor %}
</div>
```